### PR TITLE
@kanaabe => Article artworks use 'artists' array

### PR DESCRIPTION
**Pull request by eessex**
- Article artworks print names from 'artists' array by default and fallback to 'artist'
- Tests updated to use 'artists' array

### CODE REVIEW

File: components/article/test/templates.coffee [hunk 1]
```diff
@@ -207,10 +207,10 @@ describe 'article show template', ->
...
                 },
-                artist: {
+                artists: [{
                   name: "Govinda Sah 'Azad'",
                   slug: "govinda-sah-azad"
-                }
+                }]
               }
             ]
           }
```
Comment (kanaabe) [inline components/article/test/templates.coffee]: Nice, could we add an assertion to this test to make sure multiple `artists` get rendered correctly?

File: components/article/templates/mixins.jade [hunk 2]
```diff
@@ -105,6 +105,12 @@ mixin embed(section)
   .article-embed-container
     iframe(src=section.url height=section.height data-d-height=section.height data-m-height=section.mobile_height scrolling='no' frameborder='0')
 
+mixin printArtistName(artist)
+  if artist.slug
+    a(href='/artist/' + artist.slug )= artist.name
+  else
+    |= artist.name
```
Comment (kanaabe) [inline components/article/templates/mixins.jade]: 👍

File: components/article/test/templates.coffee (hunk 1 above)
Comment (eessex) [inline components/article/test/templates.coffee]: Sounds good, just added a line for this

**Comment by kanaabe** [comment]
Woot! Thanks for adding that assertion. Mergin

**Pull request merged**